src/panels/lovelace/editor/config-elements/hui-entity-card-editor.ts

```diff
@@ -100,7 +100,9 @@ export class HuiEntityCardEditor
   }
 
   private _valueChanged(ev: CustomEvent): void {
-    fireEvent(this, "config-changed", { config: ev.detail.value });
+    const config = ev.detail.value;
+    Object.keys(config).forEach((k) => config[k] === "" && delete config[k]);
+    fireEvent(this, "config-changed", { config });
```

Comment:
Isn't this done by HA Form Automatically?

Reply:
No, it doesn't remove values